app/controllers/user_journey_controller.rb

```diff
@@ -10,14 +10,17 @@ class UserJourneyController < ApplicationController
   before_action :dual_running, except: :index
   before_action :find_team_name
 
-
   def index
-    if current_user.permissions.component_management
-      @components = SpComponent.all + MsaComponent.all
+    if !helpers.idp_member?
```

Comment:
Try to avoid nesting where possible
`    return redirect_to users_path if helpers.idp_member?

    if current_user.permissions.component_management
      @components = SpComponent.all + MsaComponent.all
    else
      @components = SpComponent.for_user(current_user) + MsaComponent.for_user(current_user)
    end
    @total_certificates_expiring_soon = helpers.certificate_expiry_count(@components)`